lib/defaults.js

```diff
@@ -51,7 +51,7 @@ const generateUniquePackageOrdId = (ordNamespace, name, tag, visibility) => {
 module.exports = {
     $schema: "https://open-resource-discovery.github.io/specification/spec-v1/interfaces/Document.schema.json",
     openResourceDiscovery: OPEN_RESOURCE_DISCOVERY_VERSION,
-    policyLevel: "none",
+    policyLevels: "none",
```

Comment:
```suggestion
    policyLevels: ["none"],
```
See https://pages.github.tools.sap/CentralEngineering/open-resource-discovery-specification/spec-v1/interfaces/document#ord-document_policylevels, it is an array now